src/main/java/entities/Server.java

```diff
@@ -0,0 +1,71 @@
+package entities;
+
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.Role;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * An object that provides Topic CRUD methods for a Guild.
+ */
+public class Server {
+    private final Guild guild;
+    private final ArrayList<Topic> topics = new ArrayList<>();
+
+    public Server(Guild guild) {
+        this.guild = guild;
+    }
+
+    @Override
+    public int hashCode() {
+        return guild.getName().hashCode();
+    }
+
+    /**
+     * Constructs a Server object from a Guild's role list.
+     * @param guild The guild to build from
+     * @return A Server with all topics added
+     */
+    public static Server load(Guild guild) {
```

Comment:
I put this static method below the constructor because it acts similar to a constructor, but I'll shift it downwards to make it consistent with authbot's styling.